src/modules/colorpicker/main.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+#include <windows.h>
+
+void printColorRef(COLORREF);
+
+int main()
+{
+    HDC window = GetWindowDC(NULL);
+    while (true)
+    {
+        POINT cursorPosition;
+        if (GetCursorPos(&cursorPosition))
+        {
+            COLORREF color = GetPixel(window, cursorPosition.x, cursorPosition.y);
+            std::cout << "X: " << cursorPosition.x << "Y: " << cursorPosition.y << '\n';
+            printColorRef(color);
+        }
+    }
+}
+
+void printColorRef(COLORREF color)
+{
+    int red = static_cast<int>(GetRValue(color));
+    int green = static_cast<int>(GetGValue(color));
+    int blue = static_cast<int>(GetBValue(color));
+    std::cout << "Red: " << red << '\n'
+              << "Green: " << green << '\n'
+              << "Blue: " << blue << '\n'
+              << std::endl;
+}
```

Comment:
Should this be in the main source file? Also even though the func is straight fwd, Summary comments are good to have

Reply:
> any chance we can add in an animated gif for a visual?

![ColourPickerDemo](https://user-images.githubusercontent.com/46630768/81235356-87e1ea80-8faf-11ea-8e05-13a157838ee9.gif)

Here's a gif demoing what we have so far. The console logs the current color that the mouse is hovering on.

Reply:
Good points, we will make these changes.